### PR TITLE
Update version of gateway-graphql

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

Update `gateway-graphql` version in manifest.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements

### CODE REVIEW

**Review by lucasecdb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- `gateway-graphql` version
```
Comment (marcoskwkm) [inline CHANGELOG.md]: To what version?

**Review by marcoskwkm — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pedromtec) [reply CHANGELOG.md]: Major version.

Comment (marcoskwkm) [reply CHANGELOG.md]: I know, I'm asking you to add this information to the changelog. It is good to know that the version changed, but it is much better to know what version it was changed to.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- `gateway-graphql` major version
```
Comment (lucasecdb) [inline CHANGELOG.md]: I think @marcoskwkm meant something like

```suggestion
- `gateway-graphql` to version `1.x`.
```

Comment (marcoskwkm) [reply CHANGELOG.md]: ☝️ 
Sorry, I should have been more straightforward =P

**Pull request merged**